operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -35,6 +35,34 @@ if [ "${MOCK_WLS}" == 'true' ]; then
   exit 0
 fi
 
+# Arguments for shutdown
+export SHUTDOWN_PORT_ARG=${LOCAL_ADMIN_PORT:-${MANAGED_SERVER_PORT:-8001}}
+export SHUTDOWN_PROTOCOL_ARG=${LOCAL_ADMIN_PROTOCOL:-t3}
+export SHUTDOWN_TIMEOUT_ARG=${SHUTDOWN_TIMEOUT:-30}
+export SHUTDOWN_IGNORE_SESSIONS_ARG=${SHUTDOWN_IGNORE_SESSIONS:-false}
+export SHUTDOWN_WAIT_FOR_ALL_SESSIONS_ARG=${SHUTDOWN_WAIT_FOR_ALL_SESSIONS:-false}
+export SHUTDOWN_TYPE_ARG=${SHUTDOWN_TYPE:-Graceful}
+export SHUTDOWN_TIMEOUT_ARG=${SHUTDOWN_TIMEOUT:-30}
+
+# Calculate the wait timeout to issue "kill -9" before the pod is destroyed. 
+# Allow 3 seconds for the NFS v3 manager to detect the process destruction and release file locks.
+export SIGKILL_WAIT_TIMEOUT=$(expr $SHUTDOWN_TIMEOUT_ARG - 3)
+wait_and_kill_after_timeout(){
+  trace "Wait for ${SIGKILL_WAIT_TIMEOUT} seconds for ${SERVER_NAME} to gracefully shutdown." >> ${STOP_OUT_FILE}
+  sleep ${SIGKILL_WAIT_TIMEOUT}
+  trace "Graceful shutdown for ${SERVER_NAME} didn't finish in ${SIGKILL_WAIT_TIMEOUT} seconds, " \
```

Comment:
Minor: Consider removing the word "Graceful". IIRC, the shutdown may not be "Graceful" as the user can configure shutdown to be forced...

Reply:
Changed.